src/sankey/index.js

```diff
@@ -1,153 +1,198 @@
+      ])
       .nodeWidth(nodeWidth)
       .nodePadding(nodePadding)
-      .nodes(nodes)
-      .links(links)
-      .align(align)
-      .layout(layout);
+      .nodes(nodesCopy)
+      .links(linksCopy)
+      .nodeAlign(ALIGNMENTS[align])
+      .iterations(layout);
+    sankeyInstance(nodesCopy);
 
     const nWidth = sankeyInstance.nodeWidth();
-    const path = sankeyInstance.link();
+    const path = sankeyLinkHorizontal();
 
     return (
-      <svg height={height + margin} width={width + margin} className={`rv-sankey ${className}`}>
-        <g transform={`translate(${margin / 2}, ${margin / 2})`}>
-
-          {links.map((link, i) => (
-            <path
-              d={path(link)}
-              className="rv-sankey__link"
-              opacity={Number.isFinite(link.opacity) ? link.opacity : DEFAULT_LINK_OPACITY}
-              stroke={link.color || DEFAULT_LINK_COLOR}
-              strokeWidth={Math.max(1, link.dy)}
-              fill="none"
-              key={link.id || link.key || `link-${i}`} />
-          ))}
-
-          {nodes.map((node, i) => (
-            <g
-              transform={`translate(${node.x}, ${node.y})`}
-              className="rv-sankey__node"
-              opacity={Number.isFinite(node.opacity) ? node.opacity : DEFAULT_NODE_OPACITY}
-              key={node.id || node.key || `node-${i}`}>
-              <rect
-                onClick={() => onClick(node)}
-                onMouseOver={() => onHover(node)}
-                onMouseOut={() => onBlur(node)}
-                fill={node.color || DEFAULT_NODE_COLOR}
-                height={node.dy}
-                width={nWidth} />
-
-              {!hideLabels && node.name && (
-                <text
-                  textAnchor={node.x < width / 2 ? 'start' : 'end'}
-                  dy=".35em"
-                  x={node.x < width / 2 ? nWidth + 10 : -10}
-                  y={node.dy / 2}
-                >
-                  {node.name}
-                </text>
-              )}
-
-            </g>
-          ))}
-
-          {hasVoronoi && (
-            <Voronoi
-              className="rv-sankey__voronoi"
-              extent={[[-margin, -margin], [width + margin, height + margin]]}
-              nodes={nodes}
-              onBlur={onBlur}
-              onClick={onClick}
-              onHover={onHover}
-              x={d => d.x + d.dx / 2}
-              y={d => d.y + d.dy / 2}
+      <XYPlot
+        {...this.props}
+        yType="literal"
+        className={`rv-sankey ${className}`}>
+        {linksCopy.map((link, i) => (
+          <SankeyLink
+            style={style.links}
+            data={path(link)}
+            opacity={link.opacity || linkOpacity}
+            color={link.color}
+
+            strokeWidth={Math.max(link.width, 1)}
+            node={link}
+            nWidth={nWidth}
+            key={`link-${i}`}/>
+        ))
+        }
```

Comment:
don't you think the bracket should be on the same line as the `))`? Also I don't get the purpose of the following bracket